packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotAttachment/ChatbotAttachment.md

```diff
@@ -22,40 +23,44 @@ import ChatbotAlert from '@patternfly/virtual-assistant/dist/dynamic/ChatbotAler
 
 We are using [react-dropzone](https://react-dropzone.js.org) for opening the file dialog and handling drag and drop. It does not process files or provide any way to make HTTP requests to a server. If you need this, [react-dropzone](https://react-dropzone.js.org) suggests [filepond](https://pqina.nl/filepond/) or [uppy.io](https://uppy.io/).
 
-### Dialog for editing attachments
+### Attachment chip 
 
-```js file="./AttachmentEdit.tsx"
+When an attachment is successfully uploaded, a chip will appear in the message box. There are several chip variants that cover different attachment states, including: 
+- **Plain:** Default attachment chips, which display the filename and extension.
+- **Closeable:** Attachments that can be dismissed.
+- **Clickable:** Attachments that can be selected, typically to open file details.
+- **Loading:** Attachments that are still being uploaded. 
+
+```js file="./FileDetailsLabel.tsx"
 
 ```
 
-### Error displayed when attachment fails
+### Attachment preview
 
-```js file="./AttachmentError.tsx"
+```js file="./PreviewAttachment.tsx"
 
 ```
 
-### Details of file attached to chat conversation
+### Editable attachments
 
-The file extension on the uploaded file is reflected in the text below the file name.
+To allow users to edit an attached file, load a new code editor within the chatbot window. On this screen, users can make edits to a file and save changes if they'd like. Return users to the main chatbot window once they dismiss the editor.
```

Comment:
The user flow is typically click attachment chip > launch attachment preview -> launch code editor from attachment preview if that helps at all. You can dismiss from both. Preview is read-only. We don't actually save anything - that is up to the consumer.